client.go

```diff
@@ -742,11 +765,11 @@ type IdentitySessionIntermediateResponse = identityClient.IdentitySessionRequest
 
 // TozIDLoginRequest is used to login to a TozID account to get a ToznySDKV3 or active TozID session (future plan)
 type TozIDLoginRequest struct {
-	Username  string
-	Password  string
-	RealmName string
+	Username     string
+	Password     string
+	RealmName    string
 	APIBaseURL   string
-	LoginHandler func(response *IdentitySessionIntermediateResponse) (LoginActionData, error)
+	LoginHandler func(response *identityClient.IdentitySessionRequestResponse) (LoginActionData, error)
```

Comment:
Noting this as another reason I have a strong preference to [closing out PRs quickly](https://github.com/tozny/e3db-go/pull/56), it helps to keep PRs focused to the a smaller set of changes to review and reason about.